### DISCUSSION TITLE
An idea: an option to only show the highlight color for the active window

### DISCUSSION THREAD

**Discussion by noamraph** [valinet/ExplorerPatcher#2864] [Ideas]
I discovered that it's quite hard for me to tell at a glance which window is active. It looks like this:
![image](https://github.com/valinet/ExplorerPatcher/assets/1553464/a2126198-09b3-4f33-bbcc-880a36c0b143)

How about an option to only show the colored line below the active window, and not below other windows? So it will look like this:
![image](https://github.com/valinet/ExplorerPatcher/assets/1553464/8433d1d9-6559-42d1-a3cc-e111242aa3db)

If I manage to write a PR implementing this, what is the chance of this getting merged?

Thanks!
Noam

**Comment by pyrates999** [comment]
They look the same to me with New Discussion being the selected icon in the taskbar.

Edit: oh saw the difference.  Windows 10 has always been like this.  Would this be for windows 10 and 11 taskbars?

**Reply by noamraph** [reply]
I only mean it for the windows 10 taskbar. It seems to me that when the windows 11 taskbar is enabled, ExplorerPatcher doesn't draw the taskbar, so it's not relevant.

**Comment by Amrsatrio** [comment]
Of course that would be great for never combine users, I'd be happy to have that merged. But what about always combine? How to discern pinned icons with no open windows and non-active windows?

**Reply by pyrates999** [reply]
If you decide to add it, I would just make it an option under taskbar.

**Reply by noamraph** [reply]
I took a look at ExplorerPatcher code. I have assumed that the win10 taskbar on win11 was re-implemented, and thus that it should be quite easy to change its appearance, but from what I understand, it just activates the code by Microsoft which is still there. Am I correct? If so, I have no idea how it would be possible to remove the color lines below the buttons for non-active windows.

(I event found out from [this](https://superuser.com/a/1558107/106591) stack overflow answer that you can customize the colors using regedit. I discovered that I could change the second group of bytes and change the line color. However, the color for the active window remains the same as that for inactive windows)

**Reply by Amrsatrio** [reply]
> Am I correct?

Yes (for now). This is about to change though, as the Windows 10 taskbar is slowly being nuked in 24H2. Once the task band (open window buttons) has been reimplemented, your suggestion should be very easy to apply. My ETA is around June-July this year before 24H2 releases.

**Reply by noamraph** [reply]
@Amrsatrio Good to know! Thanks for your work!
